wire-ios-data-model/Source/ManagedObjectContext/CoreDataStack+Migration.swift

```diff
@@ -149,4 +149,9 @@ extension CoreDataStack {
         }
     }
 
+    public var cryptoboxDirectoryExists: Bool {
```

Comment:
Why is this in the Core Data Stack? Could we put it somewhere more appropriate?

Reply:
I was trying to find a class that can take this responsibility and can provide a public `cryptoboxDirectoryExists` property. `CoreDataStack` currently contains the info about `account`,`accountContainer` and `applicationContainer`.
Another idea was to use `ZMUserSession`.
I have not found any reason why ProteusService should be responsible for providing such info.

Reply:
what about a MigrationHelper struct with static methods?